boot/zephyr/boards/b_u585i_iot02a.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/ {
+	chosen {
+		zephyr,flash = &flash0;
```

Comment:
this should be in the board dts in zephyr

Reply:
it is in each board dts for zephyr, so I think I can remove it from each overlay